src/main/resources/templates/dailyStatus.html

```diff
@@ -0,0 +1,12 @@
+{#include base}
+{#title}Daily Status{/title}
+
+{#body}
+<div class="ui main container">
+    {#for entry in repositories}
```

Comment:
The output should be sorted Alphabetically 

Now I see:
```
quarkus-test-framework
quarkus-test-suite
beefy-scenarios
quarkus-extensions-combinations
```

Reply:
Done.